### DISCUSSION TITLE
想用云服务器远程云手机挂着

### DISCUSSION THREAD

**Discussion by za0001** [LmeSzinc/AzurLaneAutoScript#1210] [Q&A]
开发者模式里的USB调试已打开
通过[EasyN2N]和[Hin2n]局域网已经连接上了 结果设备脱机 有用红手指云手机的救救吗 端口写的是默认5555
![9}UNWLRUK05XX_W_~M4$CW9](https://user-images.githubusercontent.com/97282017/173037699-c45f3039-395e-4b36-b5c4-feef0bcee147.png)
端口写5554则结果是5554:                     
         由于目标计算机积极拒绝，无法连接。 (10061)

**Comment by akiziname** [comment]
估计不行，这个开销太大了，带宽和流量不一定够
有时间搞这个不如看看怎么在服务器上装模拟器直接在服务器上跑吧

**Comment by bananaheaven** [comment]
参考https://www.perfare.net/archives/1804 ，老早以前就有大佬实现了
根据我的测试，与云手机的机型有关，有些机型确实连不上，不知道什么原因，以下是我自己测试过成功的机型
红手指：KVIP
雷电云：KVIP，SVIP，BVIP
以上仅供参考，真要买的建议先买一天测试一下

**Comment by bananaheaven** [comment]
另外说下开销问题，我买的腾讯云的轻量服务器，4m带宽完全没问题，24小时挂流量一天3g左右。

**Comment by LmeSzinc** [comment]
选一家有公网adb服务的云手机厂商就好了，~~我不说是哪家，自己搜~~，把公网adb地址填进 alas 的 serial 里就可以了，简单方便，无缝对接。哦对，别忘了在新的设备上要改游戏设置，重新配meta阵容，因为它是保存在本地的

**Comment by chinac-sales** [comment]
星界云手机呀
